FlowCrypt/Core/Core.swift

```diff
@@ -236,47 +230,29 @@ final class Core: KeyDecrypter, KeyParser, CoreComposeMessageType {
     }
 
     private func call(_ endpoint: String, jsonData: Data, data: Data) async throws -> RawRes {
```

Comment:
Let's add `guard ready else { throw "Programming error: Core is not ready yet. Most likeyly startInBackgroundIfNotAlreadyRunning wasn't called first" }`